common/src/main/java/io/netty/util/AsciiString.java

```diff
@@ -1827,7 +1827,7 @@ private static byte toLowerCase(byte b) {
         return isUpperCase(b) ? (byte) (b + 32) : b;
     }
 
-    private static char toLowerCase(char c) {
+    public static char toLowerCase(char c) {
```

Comment:
Add documentation.

Reply:
@doom369 please address this one... after this is done I will merge :)

Reply:
done